src/lib/VideoPlayer.js

```diff
@@ -11,22 +11,9 @@ export default class VideoPlayer {
     this.init();
   }
 
-  createPlayButton(sketch) {
-    let playButton = sketch.createButton("Play");
-    playButton.mousePressed(() => {
-      this.video.play();
-    });
-  }
-
-  createPauseButton(sketch) {
-    let pauseButton = sketch.createButton("Pause");
-    pauseButton.mousePressed(() => {
-      this.video.pause();
-    });
-  }
-
   createPlayPauseButton(sketch) {
     let playPauseButton = sketch.createButton("Play/Pause");
+    playPauseButton.parent(sketch.select('#debug-container'))
```

Comment:
I think you'll want to do this same thing with the skip buttons, rebase and get latest?

Reply:
did you push your changes to main?